packages/backend/server/src/plugins/indexer/config.ts

```diff
@@ -58,4 +61,9 @@ defineModuleConfig('indexer', {
     default: '',
     env: ['AFFINE_INDEXER_SEARCH_PASSWORD', 'string'],
   },
+  'autoIndex.batchSize': {
+    desc: 'Number of workspaces automatically indexed per batch',
+    default: 10,
```

Comment:
Adjust it to 10 by default, and then slowly expand it according to the real situation when deploying